source/features/conversation-activity-filter.tsx

```diff
@@ -51,8 +51,7 @@ function processReview(review: HTMLElement): void {
 		return;
 	}
 
-	const shouldHideWholeReview = isWholeReviewEssentiallyResolved(review);
-	if (shouldHideWholeReview) {
+	if (isWholeReviewEssentiallyResolved(review)) {
 		review.classList.add(hiddenClassName);
 		return;
 	}
```

Comment:
Additionally, [hiding resolved threads](https://github.com/refined-github/refined-github/pull/5210/files#diff-2db094a8bff1492679cf3b912175ece902270acd21c77d1cfdd991328eb25db7R59-R61) :point_down: could be done via CSS only.